setup.py

```diff
@@ -160,6 +160,11 @@ def run(self):
             "google-cloud-storage>=1.30.0",
             "boto3>1",
         ],
+        "gateway": [
+            "fastapi",
+            "pydantic",
+            "uvicorn",
```

Comment:
We can pin or add new dependencies later if necessary.

Reply:
Are there some reasonable major version lower and upper bounds we can set?

Reply:
Can we just use the same rule as the core dependencies (using `< current_latest_version + 1`)?

Reply:
we can create `gateway-requirements.yml` if necessary.

Reply:
The current latest version of each package:

- https://pypi.org/project/fastapi/0.96.0/
- https://pypi.org/project/uvicorn/0.22.0/
- https://pypi.org/project/pydantic/2.0b2/

Reply:
Do we really need to pin these?

Reply:
Yes, it's our policy to do pinning on major versions now + compatibility tests. I think we should do the same for gateway

Reply:
For pydantic, shall we add a lower version bound (`>=1`)?

Reply:
Sounds good. I'll add `gateway-requirements.yml`.